docs/documentation.md

```diff
@@ -103,4 +102,4 @@ The toolkit provides a [Performance Optimization Guide](optimization_guide/dldt_
 You can choose to use the [OpenVINO™ Deep Learning Workbench](@ref workbench_docs_Workbench_DG_Introduction), a web-based tool that guides you through the process of converting, measuring, optimizing, and deploying models. This tool also serves as a low-effort introduction to the toolkit and provides a variety of useful interactive charts for understanding performance.
 
 ## Media Processing
```

Comment:
Section name 'Media Processing' is confusing, see another comment with change proposal.